tests/Security/TwoFactor/Provider/TwoFactorProviderInitiatorTest.php

```diff
@@ -36,7 +37,7 @@ protected function setUp(): void
 
         $this->twoFactorTokenFactory = $this->createMock(TwoFactorTokenFactory::class);
 
-        $this->initiator = new TwoFactorProviderInitiator($providerRegistry, $this->twoFactorTokenFactory);
+        $this->initiator = new TwoFactorProviderInitiator($providerRegistry, $this->twoFactorTokenFactory, new TwoFactorProviderDecider());
```

Comment:
This should be mocked to not create a dependency of the test towards the specific implementation of `TwoFactorProviderDecider`.

Reply:
I hope the new test is the way you like it.